fireplace/actions.py

```diff
@@ -705,9 +707,24 @@ class Discover(TargetedAction):
 	ARGS = ("TARGET", "CARDS")
 
 	def get_target_args(self, source, target):
+		# use hero class for Discover if not neutral (eg. Ragnaros)
+		if target.hero.data.card_class != CardClass.NEUTRAL:
```

Comment:
Comment nit -- I'd prefer keeping elif on the following line, and then placing the comments underneath the conditions, e.g.:

```python
if ...
  # comment
  ...
elif ...
  # comment
```